frontend/src/static/js/v_summary.js

```diff
@@ -681,26 +681,27 @@ window.vSummary = {
       });
     },
 
-    openTabZoomSubrange(userOrig) {
+    openTabZoomSubrange(userOrig, repo, index) {
```

Comment:
Since these lines are modified, can you help me to rename `userOrig` to `user` so that it is consistent with how authorship parameters were named?

Reply:
Okay will.

Reply:
>If I rename this, there is a const user in the function which I will have to rename too and correspondingly change it in zoom.pug too. Let me know how you want me to go about it.

Can go ahead, if it doesn't affect any functionality or readability

Reply:
@eugenepeh @yong24s @fzdy1914 Could you all recommend the new name for `const user`? As of now, I have interchanged `user` and `userOrig`

Reply:
Just a question, does `userOrig` stands for `original user`? I am unsure about this `Orig`.

Reply:
> Just a question, does `userOrig` stands for `original user`? I am unsure about this `Orig`.

This question is a hint that the variable name can be improved. Avoid abbreviations as much as possible.

Reply:
>@eugenepeh @yong24s @fzdy1914 Could you all recommend the new name for const user? As of now, I have interchanged user and userOrig

I don't think `original user` has any meaning behind it, probably just the way the developer named it, hence should be renamed to `user`.

Reply:
I think the previous developer named it as `userOrig` because he creates a shallow clone of the original user using `Object.assign({}, userOrig)`. So this is to differentiate between the original user (which should not be modified) and the cloned one (which we will be modified for the zoom tab functionality).

Reply:
>I think the previous developer named it as userOrig because he creates a shallow clone of the original user using Object.assign({}, userOrig). So this is to differentiate between the original user (which should not be modified) and the cloned one (which will be modified for the zoom tab functionality).

I see, in this case, probably should named the `userOrig` to `sourceUser` or named the cloned user to `clonedUser`. The latter is preferred because the variable names shouldn't be brought across functions.

Reply:
Okay! Moving forward by naming it as `clonedUser`